asterisk/agi/application/controllers/CallsController.php

```diff
@@ -574,6 +626,14 @@ public function addheadersAction()
                 $this->agi->setSIPHeader("X-Info-Friend", $friend->getRequestURI($exten));
                 $this->agi->setSIPHeader("X-Info-MaxCalls", 0);
             }
+            $retail = $endpoint->getRetailAccount();
+            if (!empty($retail)) {
```

Comment:
I like the second one.

I've never known where to use one or the other. We have empty($var), isset($var), is_null($var) and $var everywhere in the code :disappointed: